canvas_modules/common-canvas/src/object-model/object-model.js

```diff
@@ -1298,6 +1298,10 @@ export default class ObjectModel {
 		this.store.dispatch({ type: "SET_BOTTOM_PANEL_CONFIG", data: { config: { panelHeight: ht } } });
 	}
 
+	setRightPanelWidth(wdth) {
+		this.store.dispatch({ type: "SET_RIGHT_FLYOUT_CONFIG", data: { config: { panelWidth: wdth } } });
+	}
+
```

Comment:
If you look at the comments this is in a section for 'Bottom panel methods'.  There is a section above for 'Right flyout methods' where this code should go.  

Also in the 'Right flyout methods'  can we add a `isRightFlyoutOpen()` method to keep things consistent?